docs/quick-start.md

```diff
@@ -1,4 +1,4 @@
-# rdio-ansible-installer
+# mosaic-ansible-installer
 
 Ansible role to deploy Rubrik DatosIO RecoverX 3.0 in a single or multi-node deployment on CentOS 7.x x64 or Ubuntu 14 or 16 x64.
```

Comment:
Still has a reference to DatosIO RecoverX on line 3 of quick-start.md

Reply:
Changed to Mosaic